lib/worker/merge.js

```diff
@@ -20,25 +20,24 @@ api.merge = async ({
   worker, priorityPeers = [], witnesses = worker.consensusState.witnesses,
   basisBlockHeight, peerWitnessParentThreshold
 }) => {
-  if(witnesses.length === 0) {
+  if(witnesses.size === 0) {
     throw new TypeError('"witnesses" must be a non-empty array.');
   }
-  const witnessSet = new Set(witnesses);
```

Comment:
Same concern as previous comment around preserving the mutability of parameters.

Reply:
I'm not showing any mutations occurring on the `Set` itself in this file:

```
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'delete(' ./lib/worker/merge.js 
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'clear(' ./lib/worker/merge.js
andrewjones@Andrews-MacBook-Pro-2 bedrock-ledger-consensus-continuity % grep -inr 'add(' ./lib/worker/merge.js
./lib/worker/merge.js:324:  parentHashes.add(head.eventHash);
./lib/worker/merge.js:338:      parentHashes.add(peerHead.eventHash);
./lib/worker/merge.js:339:      peerCreatorSet.add(peerHead.creator);
./lib/worker/merge.js:347:    parentHashes.add(peerCommitmentParent.eventHash);
./lib/worker/merge.js:348:    peerCreatorSet.add(peerCommitmentParent.creator);
./lib/worker/merge.js:357:      parentHashes.add(eventHash);
./lib/worker/merge.js:391:      parentHashes.add(committedTo.eventHash);
./lib/worker/merge.js:392:      peerCreatorSet.add(committedTo.creator);
./lib/worker/merge.js:499:        pendingLocalRegularEventHashes.add(eventHash);
./lib/worker/merge.js:500:        parentHashes.add(eventHash);
./lib/worker/merge.js:516:        parentHashes.add(peerHead.eventHash);
./lib/worker/merge.js:517:        peerCreatorSet.add(peerHead.creator);
```